### DISCUSSION TITLE
Monitoring GraphQL performance

### DISCUSSION THREAD

**Discussion by raja-jamwal** [spaceuptech/space-cloud#1592] [Ideas]
Hi, I came across your product and totally loved how easy and simple it's to setup GraphQL over your DB.

However I was looking into to the documentation to find support for piping performance metrics. For example hasura (https://hasura.io), something similar to space-cloud provides integration to pipe query/mutation, perf metrics to newrelic, datadog etc or if there is a provision to export structured logs and pipe them through a data sink through a sidecar.

**Comment by YourTechBud** [comment]
That's a great idea. This feature isn't available at the moment though but should be fairly straightforward to implement
